soda/core/soda/core/api/data_source_utils.py

```diff
@@ -6,14 +8,20 @@ class DataSourceUtils:
     EXPERIMENTAL: Utility API for DataSource and connections.
     """
 
-    def __init__(self, config_path: str):
+    def __init__(self, config_path: str | None = None, config_string: str | None = None):
         """
         All the features for config files (e.g. env vars) are supported.
         :param config_path: The director path containing the data source config yaml files.
```

Comment:
Can you update the function doc string too?

Reply:
🤦 updated